evap/static/scss/components/_callouts.scss

```diff
@@ -92,3 +92,17 @@
         }
     }
 }
+
+.infobox.callout {
```

Comment:
I think this means "an element with the class `callout` inside an element with the class `infobox`". Yet, in the commit introducing the change, you change the div in `infotexts.html` to have both the `infotext` and the `callout` class. Was that intended?

Reply:
No, I think this is correct. What you describe is `.foo .bar` with a space